src/components/Road.vue

```diff
@@ -17,6 +17,7 @@
       :is-open="visibleList[index-1]"
       :adding-from-card="addingFromCard"
       :dragging-over="dragSemesterNum===index-1"
+      :hide-iap="hideIAP === 'true'"
```

Comment:
It would be clearer if hideIAP were a boolean to begin with

Reply:
It was initially a boolean but the problem is that we can't store booleans into cookies I think. Is there a workaround for this?

Reply:
I would just convert the string to a boolean when you get it out of the cookie/put it into a cookie.